ProjectSource/app-library/src/test/java/com/kayako/sdk/android/k5/common/view/CircleImageViewTest.java

```diff
@@ -0,0 +1,379 @@
+package com.kayako.sdk.android.k5.common.view;
+
+import static junit.framework.Assert.fail;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.core.IsEqual.equalTo;
+import static org.junit.Assert.assertThat;
+import static org.powermock.api.support.membermodification.MemberMatcher.method;
+import static org.powermock.api.support.membermodification.MemberModifier.suppress;
+
+import android.content.Context;
+import android.graphics.Bitmap;
+import android.graphics.ColorFilter;
+import android.graphics.Paint;
+import android.graphics.drawable.Drawable;
+import android.net.Uri;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.ImageView.ScaleType;
+import com.kayako.sdk.android.k5.R;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.Mockito;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({CircleImageView.class,ImageView.class,Paint.class,View.class})
+public class CircleImageViewTest {
+
+  private final static String INVALID_BOUNDS_MESSAGE = "adjustViewBounds not supported.";
```

Comment:
⚠️  The order of modifier specification should follow the Java standard style guide as in the proposed solution. Likewise, the field declarations should be done in the order of visibility, then by modifier. This suggestion will add the advantage that other people will be able to read your code more easily.  

**[More details](https://confluence.devfactory.com/x/VY8-Ew)**
          
## The possible solution in this case

```JAVA
private static final String INVALID_BOUNDS_MESSAGE = "adjustViewBounds not supported.";

```
 

<a id="05ef9090-4e70-11e8-8f04-8143e47b96ef" href="none" codereview-uuid-reserved-tag />